### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by neenjaw**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,2 @@
 * @exercism/maintainers-admin
+* neenjaw
```
Comment (iHiD) [inline .github/CODEOWNERS:2]: The second line will override the first. What you really want is to make you (and I think angelika so you can review each other's stuff) owners of everything, then leave the maintainers admin to be codeowners of this file.

You also need the `@` for your name

```suggestion
* @neenjaw @angelikatyborska 
.github/CODEOWNERS @exercism/maintainers-admin
```

Could you update the other repos accordingly pls? (cc @angelikatyborska)

**Comment by iHiD** [comment]
@ErikSchierboom Can you check I've fixed this properly pls :)

File: .github/CODEOWNERS (hunk 1 above)
Comment (neenjaw) [reply .github/CODEOWNERS]: Oops! Yes

**Comment by ErikSchierboom** [comment]
@iHiD It looks good to me. For the record: you're okay with us not being in the loop anymore with these updates to the tooling repos, right?

**Comment by iHiD** [comment]
Correct. Until launch, when I'll reconsider what makes most sense. For "trusted" individuals, where "trusted" means I've spent enough time chatting to them over the last 12 months to be confident in both ability and the fact they're not going to cause my bank account pain by trying to insert exploits :)

**Review by iHiD — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (angelikatyborska) [reply .github/CODEOWNERS]: Thank you for adding me to codeowners of the three Elixir tools! ❤️

**Pull request merged**